_layouts/quality_estimation.html

```diff
@@ -0,0 +1,124 @@
+---
+layout: default
+parent: Quality estimation
+---
+
+<h1>
+  {{ page.title }}
+</h1>
+<h2>
+  {{ page.description }}
+</h2>
+
+{{ content }}
+
+<hr/>
+
+<p>
+  {% if page.supported_languages.size > 0 %}
+   <strong>{{ page.title }}</strong> supports <a href="#language-support">{{ page.supported_languages | size }} languages</a>.
+  {% else %}
+    The <strong>{{ page.title }}</strong> supports an unknown number of languages.
+  {% endif %}
+</p>
+
+<p>
+  {% if page.customisation %}
+  <a href="/customisation">Customisation</a> is supported.
+  {% endif %}
+</p>
+
+
+<blockquote>
+
+  <dl>
+
+    <dt>Language support</dt>
+  {% if page.supported_languages.size == 0 %}
+    <dd><a href="#language-support">Unknown number of languages</a></dd>
+  {% else %}
+    <dd><a href="#language-support">{{ page.supported_languages | size }} languages</a></dd>
+  {% endif %}
+
+  
+  <dt>Integrations</dt>
+  {% if page.only_compatible_tms %}
+  <dd><a href="/{{ page.only_compatible_tms }}">{{ page.only_compatible_tms|capitalize }}</a></dd>
+  {% else %}
+  <dd>Any translation management system (API)</dd>
+  {% endif %}
+  
+  <dt>APIs</dt>
+  {% if page.only_compatible_mt_api %}
+  <dd><a href="/{{ page.only_compatible_mt_api }}">{{ page.only_compatible_mt_api|capitalize }}</a></dd>
+  {% else %}
+  <dd>Any machine translation API</dd>
+  {% endif %}
+</dl>
+
+  {% if page.urls.size > 0 %}
+    <hr/>
+    <ul>
+      {% for url in page.urls %}
+      <li>
+        <a href="{{ url }}" target="_blank">{{ url | remove_first: 'https://' | remove_first: 'http://' | remove_first: 'www.' | truncate: 50 }}</a>
+      </li>
+      {% endfor %}
+    </ul>
+  {% endif %}
+
+</blockquote>
+
+
+<section id="language-support">
+  <h2>Language support</h2>
+  {% if page.supported_languages.size > 0 %}
+
+    <details>
+      <summary>
+        {% if page.supported_languages_comment %}
+          {{ page.supported_languages_comment }}
+        {% endif %}
+        {% if page.supported_languages.size == 1 %}
+          <strong>1</strong> language is supported by the {{ page.title }} Company.
```

Comment:
To avoid the passive voice and adding "Company", I'd change the structure to:

´{{ page.title }} supports <strong>1</strong> language´